Description: Prevent undefinition of stat
Author: Stephen Kitt <skitt@debian.org>

MinGW-w64 redefines the stat structure, and the rpl_ redefinitions in
sys/stat.h here play havoc with that. In any case rpl_stat isn't
defined...

--- a/gnu/sys_stat.in.h
+++ b/gnu/sys_stat.in.h
@@ -574,19 +574,6 @@
 
 #if @GNULIB_STAT@
 # if @REPLACE_STAT@
-/* We can't use the object-like #define stat rpl_stat, because of
-   struct stat.  This means that rpl_stat will not be used if the user
-   does (stat)(a,b).  Oh well.  */
-#  undef stat
-#  ifdef _LARGE_FILES
-    /* With _LARGE_FILES defined, AIX (only) defines stat to stat64,
-       so we have to replace stat64() instead of stat(). */
-#   define stat stat64
-#   undef stat64
-#   define stat64(name, st) rpl_stat (name, st)
-#  else /* !_LARGE_FILES */
-#   define stat(name, st) rpl_stat (name, st)
-#  endif /* !_LARGE_FILES */
 _GL_EXTERN_C int stat (const char *name, struct stat *buf) _GL_ARG_NONNULL ((1, 2));
 # endif
 #elif defined GNULIB_POSIXCHECK
